Source/Views/HeaderView.swift

```diff
@@ -12,6 +12,9 @@ open class HeaderView: UIView {
       attributes: LightboxConfig.CloseButton.textAttributes)
 
     let button = UIButton(type: .system)
```

Comment:
another nit if its easy: ideally haptics when close is clicked if its easy

Reply:
Turns out it is easy! Done, thanks.